docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -159,12 +159,12 @@ performed by [tcping](https://github.com/zhengxiaowai/tcping)
 
 | Metrics                                      | Unit     | Description                                                                          |
 | -------------------------------------------- | -------- | ------------------------------------------------------------------------------------ |
-| tcp-connectivity/Successed_${hostname/ip}    | count    | successed times of tcp connections between current node and other nodes              |
-| tcp-connectivity/Failed_${hostname/ip}       | count    | failed times of tcp connections between current node and other nodes                 |
-| tcp-connectivity/Success_Rate_${hostname/ip} | count    | success rate(successed/total) of tcp connection between current node and other nodes |
-| tcp-connectivity/Minimum_${hostname/ip}      | time(ms) | mininum latency of tcp connections between current node and other nodes              |
-| tcp-connectivity/Maximum_${hostname/ip}      | time(ms) | maximum latency of tcp connections between current node and other nodes              |
-| tcp-connectivity/Average_${hostname/ip}      | time(ms) | average latency of tcp connections between current node and other nodes              |
+| tcp-connectivity/successed_${hostname/ip}    | count    | successed times of tcp connections between current node and other nodes              |
+| tcp-connectivity/failed_${hostname/ip}       | count    | failed times of tcp connections between current node and other nodes                 |
+| tcp-connectivity/success_rate_${hostname/ip} | count    | success rate(successed/total) of tcp connection between current node and other nodes |
+| tcp-connectivity/minimum_${hostname/ip}_time | time(ms) | mininum latency of tcp connections between current node and other nodes              |
+| tcp-connectivity/maximum_${hostname/ip}_time | time(ms) | maximum latency of tcp connections between current node and other nodes              |
+| tcp-connectivity/average_${hostname/ip}_time | time(ms) | average latency of tcp connections between current node and other nodes              |
```

Comment:
ib loopback uses "avg" instead of "average", maybe change to min, max, avg here?

Reply:
ok, updated